http-add-on/values.yaml

```
@@ -17,6 +15,7 @@ operator:
   port: 8443
 
 images:
+  tag: latest
```

Since we'd prefer it to default to the app version, we should just leave this empty here
```suggestion
  tag:
```